docroot/themes/humsci/humsci_basic/readme.md

```diff
@@ -0,0 +1,54 @@
+# Stanford Basic theme
+
+Stanford Basic is used a base for all future H&S themes. It integrates selectively with Decanter Sass, styles and 
+patterns.
+
+## Requirements
+
+- Node 10.15+
+
+## Getting Started
+
+This theme contains it's own node module dependencies and build system separate from the root project. All commands 
+should be run from this theme's directory.
+
+- `npm install` - Install all node dependencies
+
+## Builds 
+
+Frontend assets are built using the Grunt task runner, but using npm scripts as shortcuts. Assets are compiled to a 
+`dist/` directory with a stylesheet generated for each sub-theme.
+
+- `npm run build:sass` - Compile Sass for production
+- `npm run watch` - Compile a build and watch for changes
+
+## Testing
+
+- `npm test` - Run linting and Sass tests
+
+### Sass True
+
+We use the [Sass True](https://github.com/oddbird/true) testing framework to test our Sass function and mixins.
+
+### Linting
+
+We use [stylelint](https://stylelint.io/) to lint all of our Sass code to maintain a consistent code style.
+Our linting rules use the [Sparkbox Stylelint Config](https://github.com/sparkbox/stylelint-config-sparkbox) as a base 
+for our linting rules.
+
+## Decanter Integration
+
+This theme aims to **partially** integrate [Decanter](https://github.com/SU-SWS/decanter). Instead of rendering all the
+styles generated by Decanter we:
+
+- Import variables, function and mixins
+- Compile various helpers classes
+- Compile specific components such as Brand Bar, Logo, Lockup and Footer
+
+## Sub-themes
+
+There are currently 3 sub-themes based on Humsci Basic
```

Comment:
These themes reference humsci_basic css builds on a per-theme basis in their x.libraries.yml